workloads/network.yml

```diff
@@ -0,0 +1,147 @@
+---
+#
+# Runs network benchmarks on an existing RHCOS cluster
+#
+
+- name: Runs network benchmarks on a RHCOS cluster
+  hosts: orchestration
+  gather_facts: true
+  remote_user: "{{orchestration_user}}"
+  vars_files:
+    - vars/network.yml
+  vars:
+    workload_job: "network"
+  tasks:
+    - name: Create scale-ci-tooling directory
+      file:
+        path: "{{ansible_user_dir}}/scale-ci-tooling"
+        state: directory
+
+    - name: Copy workload files
+      copy:
+        src: "{{item.src}}"
+        dest: "{{item.dest}}"
+      with_items:
+        - src: scale-ci-tooling-ns.yml
+          dest: "{{ansible_user_dir}}/scale-ci-tooling/scale-ci-tooling-ns.yml"
+        - src: workload-network-script-cm.yml
+          dest: "{{ansible_user_dir}}/scale-ci-tooling/workload-network-script-cm.yml"
+
+    - name: Slurp kubeconfig file
+      slurp:
+        src: "{{kubeconfig_file}}"
+      register: kubeconfig_file_slurp
+
+    - name: Slurp ssh private key file
+      slurp:
+        src: "{{pbench_ssh_private_key_file}}"
+      register: pbench_ssh_private_key_file_slurp
+
+    - name: Slurp ssh public key file
+      slurp:
+        src: "{{pbench_ssh_public_key_file}}"
+      register: pbench_ssh_public_key_file_slurp
+
+    - name: Remove network test labels from nodes
+      shell: |
+        oc label nodes -l node-role.kubernetes.io/worker= uperf-
+
+    - name: Label worker nodes with pbench agent for network tests
+      shell: |
+        oc get nodes -l pbench_agent=true,node-role.kubernetes.io/worker= | grep worker | head -n 1 | awk '{print $1}' | xargs -I % oc label nodes % uperf=client
+        oc get nodes --show-labels -l pbench_agent=true,node-role.kubernetes.io/worker= | grep worker | grep -v uperf=client | head -n 1 | awk '{print $1}' | xargs -I % oc label nodes % uperf=server
+      when: enable_pbench_agents|bool
+
+    - name: Label worker nodes when there are no pbench agents for network tests
+      shell: |
+        oc get nodes -l node-role.kubernetes.io/worker= | grep worker | head -n 1 | awk '{print $1}' | xargs -I % oc label nodes % uperf=client
```

Comment:
Not important, feel free to ignore, but can be simplified with awk doing the grep.  Also suggest using `--no-headers` , but other tests have the same issue.

Reply:
Good point, I will fix this playbook and feel free to fix the others otherwise I will put together a pr to do so.

Reply:
I'll create a PR do this.